tests/release/const.go

```diff
@@ -0,0 +1,23 @@
+package release
```

Comment:
I'd say this file should contain only shared constants, not those that are specific to a single file.

Reply:
Extracted release failure tests constants from shared const.go into release-failures.go file

Reply:
Extracted release failure tests related constants into release-failures.go